Restructure and improve the docs (#679)
dylibso/chicory: docs/docs/usage/6-bom.md

---
sidebar_position: 6
sidebar_label: BOM
title: BOM
---

> # Bill Of Material

Reviewer: ```suggestion
# Bill of Materials
```

> To keep the versions of different Chicory's artifact aligned in your project you can use the provided [Maven BOM file](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms).

Reviewer: ```suggestion
To keep the versions of different Chicory artifacts aligned in your project you can use the provided
[Maven BOM file](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms).
```

Import it in the `dependencyManagement` section of your `pom.xml`:

```xml
<dependencyManagement>
    <dependencies>
        <dependency>
            <groupId>com.dylibso.chicory</groupId>
            <artifactId>bom</artifactId>

>             <version>${chicory.version}</version>

Reviewer: Missing the important part
```suggestion
            <version>${chicory.version}</version>
            <type>pom</type>
            <scope>import</scope>
```

        </dependency>
    </dependencies>
</dependencyManagement>
```

> and you can use any Chicory dependency without setting the version number again in the build:

Reviewer: ```suggestion
and you can use any Chicory dependency without declaring the version number again in the build:
```
